### DISCUSSION TITLE
Should we move the `Arcus.EventGrid.Testing` to this repo?

### DISCUSSION THREAD

**Discussion by stijnmoreels** [arcus-azure/arcus.testing#47] [Q&A] [answered]
The `Arcus.EventGrid.Testing` contains functionality that we use throughout our own repo's, just like the `Arcus.Testing` repo. The question is: is the EventGrid-repo something we support as a public testing capability, or something that we choose for our own functionality?

In case of the later, it's maybe an opportunity to move it here.

**Comment by tomkerkhove** [comment, marked as answer]
I would personally keep it where it is because that feels best to me given the focus is on Event Grid itself; but that's just my opinion.
